### PR TITLE
added new C programs 

### PR DESCRIPTION
**Pull request by theanjali01**

### CODE REVIEW

File: 01_arg_and_return.exe
Comment (sgrdev) [inline 01_arg_and_return.exe:1]: please gitignore executables

File: pattern/05_zeroandone
Comment (sgrdev) [inline pattern/05_zeroandone:1]: this file is missing .c extension

**Review by sgrdev — CHANGES_REQUESTED** [review]
please fix the issues i mentioned

File: 01_arg_and_return.exe
Comment (twlite) [reply 01_arg_and_return.exe:1]: @theanjali01 pls remove this file 😓

File: controlStatement/05_switch-case.c
Comment (twlite) [inline controlStatement/05_switch-case.c:1]: why is it commented?

**Review by twlite — CHANGES_REQUESTED** [review]

**Review by twlite — APPROVED** [review]

**Pull request merged**